packages/components/src/circular-option-picker/stories/index.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import CircularOptionPicker from '..';
+
+const meta: ComponentMeta< typeof CircularOptionPicker > = {
+	title: 'Components/CircularOptionPicker',
+	component: CircularOptionPicker,
+	parameters: {
+		actions: { argTypesRegex: '^on.*' },
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
+	},
+};
```

Comment:
I noticed the `actions` control in Storybook is currently displaying a gnarly object that isn't really useful... I wasn't 100% sure what caused that, but we can avoid it by setting that control type to `null`. What do you think?

Reply:
✅ e1d2aaeb2c